js/fizzbuzz.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
+
+var FizzBuzz = (function() {
+
+  var array = [];
+  var fizzBuzz = function (firstWord, secondWord) {
+
+    if (firstWord !== '') {
+      this.firstWord = firstWord;
+    }
+    else {
+      this.firstWord = 'Fizz';
+    }
+
+    if (secondWord !== '') {
+      this.secondWord = secondWord;
+    }
+    else {
+      this.secondWord = 'Buzz';
+    }
+
+    console.log(this.firstWord);
+
+  }
+
+  fizzBuzz.prototype.readArray = function(lowNumber, highNumber) {
+
+    for (var i = 0; i < (highNumber - lowNumber + 1); i++) {
+
+      array[i] = lowNumber + i;
+        if (array[i] === 0) {
+          array[i] = 0;
+        }
+        else if (array[i] % 3 === 0 && array[i] % 5 === 0){
+            array[i] = this.firstWord + this.secondWord
+        }
+        else if (array[i] % 5 === 0){
+            array[i] = this.secondWord
+        }
+        else if (array[i] % 3 === 0){
+            array[i] = (this.firstWord)
+        }
+
+      }
+    console.log(array);
+    return array;
+  };
+
+  fizzBuzz.prototype.writeArray = function (el) {
+
+    var fizzBuzzData = document.createElement('ul');
+
+    for (var j = 0; j < array.length; j++) {
```

Comment:
Although it's totally valid, is there a reason you went with `j` here instead of `i`? `i` should be properly scoped to both the `readArray` function as well as the `for` loop in which it's being defined/invoked, so you shouldn't have any collisions.